src/ontology/agro-edit.owl

```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
 EquivalentClasses(<http://purl.obolibrary.org/obo/AGRO_00010155> <http://purl.obolibrary.org/obo/CHEBI_33287>)
 SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010155> <http://purl.obolibrary.org/obo/BFO_0000023>)
 
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010156> (live fence)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "\"Harvey, C.A., Villanueva, C., Villacís, J., Chacón, M., Muñoz, D., López, M., Ibrahim, M., Gómez, R., Taylor, R., Martinez, J. and Navas, A., 2005. Contribution of live fences to the ecological integrity of agricultural landscapes. Agriculture, ecosystems & environment, 111(1-4), pp.200-230. ecological integrity of agricultural landscapes\"") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010156> "Fence established by planting large cuttings, that easily produce roots and on which several strings of wire are attached with the obvious purpose of keeping livestock in or out."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010156> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010156> "2021-12-24T00:21:23Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/AGRO_00010156> "fenceline planting")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010156> "live fence"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010156> <http://purl.obolibrary.org/obo/ENVO_01000468>)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010156> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000056> <http://purl.obolibrary.org/obo/AGRO_00000584>))
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010157> (boundary planting)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "Technical Specification Boundary Planting. https://takingroot.org/wp-content/uploads/2015/04/takingroot-boundary-planting.pdf") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010157> "A planting process where trees or hedges are planted along boundaries to delimitate property, agricultural fields, pastures, roads or any other place where fences might be located."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010157> "https://orcid.org/0000-0003-3985-4911")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010157> "2021-12-27T16:54:13Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010157> "boundary planting"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010157> <http://purl.obolibrary.org/obo/AGRO_00000231>)
```

Comment:
I wonder if this term should go under ''live fence'' class. I have found that exact syn are:  barrier planting, Trees on boundaries and live fence. 
The term is a bit tricky as it includs a verb. But I think the term does not describe a process but the finality of this process (based on the other term in ERA list), and is therefore a continuant.  

Definition: Live fence made of trees. Boundary planting are used to delimitate property, agricultural fields, pastures, roads or any other place where fences might be located.

Source: Baker, K., Baumann, D., Michaud, J.S., van Mossel-Forrester, B., 2014. Technical Specification Boundary Planting. Registered Plan Vivo Project: The CommuniTree Carbon Project. 64p. URL: https://takingroot.org/wp-content/uploads/2015/04/takingroot-boundary-planting.pdf